DS/DSassignment1/ArrayList1.java

```diff
@@ -0,0 +1,203 @@
+import java.util.Arrays;
+
+public class ArrayList1 {
+	
+	public static int defaultSize = 10;
+	public static int size=0;
+	public static Object data[] = new Object[defaultSize];
+	
+	public static int index;
+	
+	
+	public  void add(Object value){
+		if(size == data.length ){
+			increaseCapacity();
+		}
+		data[size++] = value;
+	
+		
+	}
+	
+	public void merge(){
+		if((ArrayList2.size + size) >= data.length ){
+			increaseCapacity();
+		}
+		for(int i = 0; i < ArrayList2.size;i++)
+		add(ArrayList2.newData[i]) ;	
+		System.out.println("both list merged");
+		
+	}
+	public static void increaseCapacity(){
```

Comment:
ACTION: shouldn't this method be private? Make a habit to keep things private by default and think if public access is required for it. The advantage of this is if something is wrongly made private, while writing a test class u will know if some method/field needed to be public/default.